stock-usecase/src/main/java/org/dhis2/android/rtsm/ui/managestock/ManageStockViewModel.kt

```diff
@@ -321,7 +321,7 @@ class ManageStockViewModel @Inject constructor(
                     is ValidationResult.Error -> {
                         addItem(
                             item = stockItem,
-                            qty = cell.value?.ifEmpty { "0" },
+                            qty = cell.value,
```

Comment:
did you tested that his change (wich I consider correct), allow the user to move on next between empty cells?

Reply:
yes